src/index.html

```diff
@@ -4,14 +4,191 @@
+      onsubmit="onSubmit()"
+    >
+      <script type="text/javascript" src="./main.js"></script>
+      
+      <fieldset class = "form-block">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input 
+            type="text" 
+            name="surname" 
+            id="surname" 
+            autocomplete="off"
+            required
+          >
+        </div>
+        
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input 
+            type = "text" 
+            name = "name" 
+            id="name" 
+            autocomplete="off"
+            required
+          >
+        </div>
+  
+        <div class="form-field">
+          <label for="age">How old are you?</label>
+          <input 
+            type="number" 
+            name="age" 
+            id="age" 
+            value="12"
+            min="12"
+            max="100"
+          >
+        </div>
+  
+        <div class="form-field">
+          <label for="date">Full date of birth:</label> 
+          <input 
+            type="date" 
+            name="date-birth" 
+            id="date"
+          >
+        </div>
+  
+        <div class="form-field">
+          <label for="accept">I accept the term of agreement:</label>
+          <input 
+            type="checkbox" 
+            name="accept" 
+            id="accept" 
+            required
+          >
+        </div>
+      </fieldset>
+      
+      <fieldset class="form-block">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail: for='email'</label> 
+          <input 
+            type = "email" 
+            name = "email" 
+            id="email" 
+            placeholder="email@example.com"
+          >
+        </div>
+  
+        <div class="form-field">
+          <label for="password">Password: </label>
+          <input 
+            type="password" 
+            name="password" 
+            id="password" 
+            required 
+            minlength="5"
+            maxlength="20"
+          >
+        </div>
+      </fieldset>
+      
+      <fieldset class="form-block">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input 
+              type="radio" 
+              name="love-cats" 
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio" 
+              name="love-cats" 
+              id="love-cats"
+              value="no"
+            >
+            No
+          </label>
+
+        </div>
+  
+        <div class="form-field">
+          <label for="favorite-color">What is you favorite color?</label> 
+          <input 
+            type = "color" 
+            name = "favorite-color" id="favorite-color"
```

Comment:
```suggestion
            type="color" 
            name="favorite-color"
            id="favorite-color"
```
Каждый атрибут с новой строки, и без отступов вокруг `=`